static/theme-flex/script.js

```diff
@@ -271,7 +284,7 @@ var getUrlParameter = function getUrlParameter(sPageURL) {
 };
 
 // Burger icon click toggle header menu on small devices
-$('.burger').on('click', function(e){
-    $('#shortcuts').toggleClass("responsive") ;
+$('.burger').on('click', function (e) {
+    $('aside').toggleClass("show-nav");
```

Comment:
This is the key change in this file - other changes are just from the pretty formatter. 

switches the hamburger icon to toggling a class on the `aside` instead of the `#shortcuts`, and changes the name of the class that gets toggled.